packages/console/src/consts/logs.ts

```diff
@@ -22,6 +22,12 @@ export const auditLogEventTitle: Record<string, Optional<string>> &
   'Interaction.ForgotPassword.Profile.Update': 'Patch update forgot-password interaction profile',
   'Interaction.ForgotPassword.Submit': 'Submit forgot-password interaction',
   'Interaction.ForgotPassword.Update': 'Update forgot-password interaction',
+  'Interaction.ForgotPassword.BindMfa.TOTP.Create': undefined,
```

Comment:
Let the copilot generate those human-readable names?

Reply:
@xiaoyijun leave it to you.